### DISCUSSION TITLE
blog/posts/serverless-north-east-13-2023

### DISCUSSION THREAD

**Discussion by giscus** [AlexHedley/blog#10] [General]
# blog/posts/serverless-north-east-13-2023

Serverless North East

https://alexhedley.com/blog/posts/serverless-north-east-13-2023

**Comment by AlexHedley** [comment]
- [Introducing EventNet: a tool for testing EventBridge events](https://medium.com/on-tech-by-leighton/introducing-eventnet-a-tool-for-testing-eventbridge-events-49357d364d6e)
- [EventNet](https://github.com/leighton-digital/eventnet)
- [EventNet Example](https://github.com/stevemorland/eventnet-example)

- https://www.linkedin.com/posts/steve-morland_introducing-eventnet-a-tool-for-testing-activity-7140731979255193600-CeqT
